core/pom.xml

```diff
@@ -116,6 +116,11 @@
             <classifier>nohive</classifier>
         </dependency>
 
+        <dependency>
+            <groupId>com.google.guava</groupId>
+            <artifactId>guava</artifactId>
+        </dependency>
```

Comment:
Now the Flink/Trino module also has its own declared Guava dependency. If the dependency is declared in the core, it seems that the dependencies of other modules can be deleted.